web-client/src/ustc-ui/Tabs/Tabs.jsx

```diff
@@ -88,6 +89,11 @@ const renderTabFactory = ({
               <div className="icon-tab-notification-exclamation">!</div>
             </div>
           )}
+          {draftCount !== '0' && (
```

Comment:
I'm curious why draftCount is a string with a numeric value.  Shouldn't this just be a javascript number?

Reply:
We will create a DEVEX task to address this feedback!

Reply:
Just a suggestion, maybe instead of DEVEX this could be done in the DoD (definition of done) for this story?